[2025-07-16T17:19:54.167+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-16T17:19:54.282+0000] {taskinstance.py:2604} INFO - Dependencies not met for <TaskInstance: bank_reviews_etl_and_analysis_pipeline.scrape_and_insert_data_to_json scheduled__2025-06-29T00:00:00+00:00 [skipped]>, dependency 'Task Instance State' FAILED: Task is in the 'skipped' state.
[2025-07-16T17:19:54.351+0000] {local_task_job_runner.py:166} INFO - Task is not able to be run
[2025-07-16T17:30:02.026+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-16T17:30:03.363+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bank_reviews_etl_and_analysis_pipeline.scrape_and_insert_data_to_json scheduled__2025-06-29T00:00:00+00:00 [queued]>
[2025-07-16T17:30:03.397+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bank_reviews_etl_and_analysis_pipeline.scrape_and_insert_data_to_json scheduled__2025-06-29T00:00:00+00:00 [queued]>
[2025-07-16T17:30:03.406+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-07-16T17:30:03.476+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): scrape_and_insert_data_to_json> on 2025-06-29 00:00:00+00:00
[2025-07-16T17:30:03.494+0000] {standard_task_runner.py:72} INFO - Started process 424 to run task
[2025-07-16T17:30:03.510+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'bank_reviews_etl_and_analysis_pipeline', 'scrape_and_insert_data_to_json', 'scheduled__2025-06-29T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/dag_load_reviews.py', '--cfg-path', '/tmp/tmpym9p4v0e']
[2025-07-16T17:30:03.517+0000] {standard_task_runner.py:105} INFO - Job 10: Subtask scrape_and_insert_data_to_json
[2025-07-16T17:30:03.882+0000] {task_command.py:467} INFO - Running <TaskInstance: bank_reviews_etl_and_analysis_pipeline.scrape_and_insert_data_to_json scheduled__2025-06-29T00:00:00+00:00 [running]> on host 569ba507bc01
[2025-07-16T17:30:05.075+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohammed' AIRFLOW_CTX_DAG_ID='bank_reviews_etl_and_analysis_pipeline' AIRFLOW_CTX_TASK_ID='scrape_and_insert_data_to_json' AIRFLOW_CTX_EXECUTION_DATE='2025-06-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-29T00:00:00+00:00'
[2025-07-16T17:30:05.099+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-07-16T17:30:05.101+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-07-16T17:30:05.103+0000] {logging_mixin.py:190} INFO - Current task name:scrape_and_insert_data_to_json state:running start_date:2025-07-16 17:30:03.370537+00:00
[2025-07-16T17:30:05.106+0000] {logging_mixin.py:190} INFO - Dag name:bank_reviews_etl_and_analysis_pipeline and current dag run status:running
[2025-07-16T17:30:05.120+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-16T17:30:05.135+0000] {logger.py:11} INFO - ====== WebDriver manager ======
[2025-07-16T17:30:08.485+0000] {logger.py:11} INFO - Get LATEST chromedriver version for google-chrome
[2025-07-16T17:30:13.397+0000] {logger.py:11} INFO - Get LATEST chromedriver version for google-chrome
[2025-07-16T17:30:22.360+0000] {logger.py:11} INFO - Driver [/home/***/.wdm/drivers/chromedriver/linux64/138.0.7204.157/chromedriver-linux64/chromedriver] found in cache
[2025-07-16T17:30:24.384+0000] {insert_data_to_json.py:20} INFO - ‚ùå Erreur lors du scraping : Message: session not created: probably user data directory is already in use, please specify a unique value for --user-data-dir argument, or don't use --user-data-dir
Stacktrace:
#0 0x55d9a6d047da <unknown>
#1 0x55d9a67a92e0 <unknown>
#2 0x55d9a67e37bd <unknown>
#3 0x55d9a67dead7 <unknown>
#4 0x55d9a682ee9e <unknown>
#5 0x55d9a682e466 <unknown>
#6 0x55d9a6820683 <unknown>
#7 0x55d9a67ecb5b <unknown>
#8 0x55d9a67edf31 <unknown>
#9 0x55d9a6cc972b <unknown>
#10 0x55d9a6ccd534 <unknown>
#11 0x55d9a6cb0229 <unknown>
#12 0x55d9a6cce0d8 <unknown>
#13 0x55d9a6c9461f <unknown>
#14 0x55d9a6cf1dd8 <unknown>
#15 0x55d9a6cf1fb6 <unknown>
#16 0x55d9a6d03af6 <unknown>
#17 0x7fd1be6371c4 <unknown>

[2025-07-16T17:30:24.406+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-16T17:30:24.500+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-16T17:30:24.503+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=bank_reviews_etl_and_analysis_pipeline, task_id=scrape_and_insert_data_to_json, run_id=scheduled__2025-06-29T00:00:00+00:00, execution_date=20250629T000000, start_date=20250716T173003, end_date=20250716T173024
[2025-07-16T17:30:24.666+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-07-16T17:30:24.672+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-07-16T17:30:24.683+0000] {logging_mixin.py:190} INFO - Dag name:bank_reviews_etl_and_analysis_pipeline queued_at:2025-07-16 17:28:11.399702+00:00
[2025-07-16T17:30:24.687+0000] {logging_mixin.py:190} INFO - Task hostname:569ba507bc01 operator:PythonOperator
[2025-07-16T17:30:24.753+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-16T17:30:24.904+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-16T17:30:24.913+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
